docs/products/kafka/howto/configure-with-kafka-cli.rst

```diff
@@ -0,0 +1,53 @@
+Manage configurations with Apache Kafka® CLI tools
+==================================================
+
+ZooKeeper is an important part of Apache Kafka® since it tracks the status of the cluster and offers visibility its the current state. Moreover, by interacting with ZooKeeper, you're able to change any Apache Kafka parameter.
+
+Aiven for Apache Kafka® is a fully managed service, therefore, to guarantee the service stability, good maintenance and functionality, we do not allow direct access to ZooKeeper. Instead, we monitor and manage it using the Aiven platform to make sure that we can react as quickly as possible if any issues occur.
+
+Most of the changes usually performed via ZooKeeper are available via the :doc:`advanced configuration flags in the Aiven web console <use-zookeeper>` or via the :doc:`Aiven CLI </docs/tools/cli>`. However, you may want to use the client tools included in the `Apache Kafka binaries <https://kafka.apache.org/downloads>`_ to manage the configurations for your services. 
```

Comment:
Could we start with this part? Aiven offers the tools you need to manage your Kafka. To keep the managed service awesome, direct ZooKeeper access isn't available, but our tooling gives you the access that you need - whether your Kafka version has Zookeeper in it or not.

For all the stuff that doesn't need Zookeeper, you can use the apache binaries in the normal way, see example below.

Reply:
Good one